<launch>


<!-- Change the robot id here too! -->
  <group ns="robot_1">

    <!-- Don't forger to change the robot ID in the name space! -->
    <param name="local_robot_id" type="int" value="1" />
    <param name="other_robot_id" type="int" value="1" />

    <remap from="left/camera_info" to="/kitti/camera_color_left/camera_info"/>
    <remap from="right/camera_info" to="/kitti/camera_color_right/camera_info"/>
    <remap from="rgb/image_rect" to="/kitti/camera_color_left/image_rect"/>
    <remap from="left/image_rect" to="/kitti/camera_color_left/image_rect"/>
    <remap from="right/image_rect" to="/kitti/camera_color_right/image_rect"/>
    <remap from="left/camera_info" to="/kitti/camera_color_left/camera_info"/>
    <remap from="right/camera_info" to="/kitti/camera_color_right/camera_info"/>
    <node pkg="multi_robot_separators" type="stereo_cam_geometric_tools" name="stereo_cam_geometric_tools"/>
    <node pkg="multi_robot_separators" type="find_separators.py" name="find_separators"/>
    <node pkg="multi_robot_separators" type="factor_graph" name="factor_graph"/>
    <node pkg="multi_robot_separators" type="communication" name="communication"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 base_link camera_color_left" />
    <node pkg="rtabmap_ros" type="stereo_odometry" name="stereo_odometry"/>
  </group>
</launch>